tools/isledecomp/isledecomp/types.py

```diff
@@ -1,4 +1,5 @@
 """Types shared by other modules"""
+
```

Comment:
We should see whether pylint can check for newlines after the docstring. I'm sure I/we were not consistent about this, but FWIW [PEP 257](https://peps.python.org/pep-0257/) rules against a blank line.

Reply:
Quite the contrary, `black` actually adds that empty line back, at least the version `24.4.2` on Python 3.12 that I am using. To me, it looks like PEP 257 specifically discusses docstrings in functions.

Furthermore, `black` actually wants to add a lot of these blank lines to files I have not touched. I wanted to do submit this in a separate PR because this one is extremely large already.

Reply:
On reviewing this again, it seems like I have commited this change by accident. Will revert it since this it belongs into the other PR.